libsolidity/ast/Types.cpp

```diff
@@ -2626,6 +2626,7 @@ string FunctionType::richIdentifier() const
 	case Kind::ABIEncodeWithSelector: id += "abiencodewithselector"; break;
 	case Kind::ABIEncodeWithSignature: id += "abiencodewithsignature"; break;
 	case Kind::ABIDecode: id += "abidecode"; break;
+	case Kind::MetaType: id += "metaType"; break;
```

Comment:
Hm... none of the other uses camelCase, but they all use lower case... I'd be fine with it, but it *is* a tiny bit inconsistent :-).